docs/Developers/Newcomers_Guide.rst

```diff
@@ -87,7 +87,23 @@ Step 1. Meet the Community!
 
 To get started, the first step is to meet the community. We use gitter to
 communicate, and there the helpful community will guide you.
-Join us at `coala gitter <https://coala.io/chat>`_.
+Gitter is instant messaging service used by developers and users of GitHub.
```

Comment:
@userzimmermann Could explain about GitLab integration since I am new to coala development. I am working on this issue.